src/test/java/org/gridsuite/modification/server/EmbeddedElasticsearch.java

```diff
@@ -24,9 +25,12 @@
 
     private static ElasticsearchContainer elasticsearchContainer;
 
+    @Value("${spring.data.elasticsearch.enabled:false}")
+    boolean elasticsearchEnabled;
+
     @PostConstruct
     public void postConstruct() {
-        if (elasticsearchContainer != null) {
+        if (elasticsearchContainer != null || !elasticsearchEnabled) {
```

Comment:
What if `!elasticsearchEnabled` **AND** `elasticsearchContainer != null` ? Maybe it should be ensured the elasticsearchContainer is stopped in this case ?

Reply:
Only one embedded elasticsearch is started for all test classes (static attribute)
And all EmbeddedElasticsearch beans are destroyed in the end of all test classes